appveyor.yml

```diff
@@ -0,0 +1,35 @@
+
+os: Visual Studio 2015
+
+environment:
+  matrix:
+    - ARCH:                x64
+      D_COMPILER:          dmd
+      D_VERSION:           2.071.2
+      C_COMPILER:          MSVC
+
+artifacts:
+  - path: src/dmd.exe
+    name: dmd
+  - path: phobos64.lib
+    name: phobos
+
+init:
+  - git config --global core.autocrlf input
+
+build_script:
+  - cd c:/projects/
+  - call "c:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64
+  - bash --version
+  - appveyor DownloadFile "https://ftp.gnu.org/gnu/make/make-4.2.tar.gz" -FileName make.tar.gz
```

Comment:
While I would love to use GNUmake on Windows, the sad reality is that DMD ships with DM make and thus that's the only make binary we can expect people to have on Windows. @CyberShadow and I have tried to push for changing the status quo in the past, but so far without any real progress :/

Reply:
@wilzbach 

As mentioned, it's required to run the tests. I'd be fine with just using GNU make for all the make scripts. It's not fun using two different make systems who's executables are both named "make". I'd much rather have one make system, then GNU make can be provided with the releases instead of DM's make.

Fixing the make situation would also be nice. Windows has the most amount of make configurations but the least amount of test servers. It's not ideal at all, this pull request was to at the very least get win64.mak tested as it isn't being run as part of any tests at all.

Reply:
We already have a go for GNU make on Windows @wilzbach, just needs someone to implement the transition.

Reply:
How about caching that installation?
https://www.appveyor.com/docs/build-cache/